Add option merge

balooval opened the pull request:
Option "merge" would merge all features into single index.html file.

Commit:
Add option merge

Option "merge" would merge all features into single index.html file.

Commit:
Add "merge" option documentation to README

matthiasnoback commented on src/FeatureExporter.php:
```diff
@@ -85,6 +86,8 @@ public function exportDirectory(
         if (is_string($tag)) {
             $this->exportAllFeaturesToSingleFile($features, $targetDirectory, $tag, $stylesheet, $reformatHtml);
+        } elseif ($merge) {
+            $this->exportAllFeaturesToSingleFile($features, $targetDirectory, 'index', $stylesheet, $reformatHtml);
```
If you want to merge features tagged "index", you can just provide `--tag index` as CLI option. I understand you want to add the CLI option to merge anyway, but I'd rather not "abuse" the tag argument for that. Maybe this can be implemented in a more generic way?

balooval replied:
What I want is got a single file without having to use tagged features. I pushed a new commit, hope it will be more like as you are thinking of.

(first contribution to github, please be indulgent)

Commit:
Join "tag" and "merge" usage for single file output

Rename some variables to be more expressive.
"tag" param is mainly to filter features. Used to determine output file name if no "merge" option provided.
